src/components/navigation/z-app-header/index.tsx

```diff
@@ -0,0 +1,16 @@
+import { Component, h } from '@stencil/core';
+
+@Component({
+  tag: 'z-app-header',
+  styleUrl: 'styles.css',
+  shadow: true
+})
+
+export class ZAppHeader {
+  render() {
+    return [
+      <slot name="title"></slot>,
```

Comment:
In this case, since the slotted element is always styled in the same way, regardless of whether it is an `h1` or other, it would not be better to avoid using slots, passing a `title` and a `subtitle` prop and rendering the strings in an `h1` and a `h2` internally composed? this would avoid the possibility of rendering an unwanted tag (a `div` instead of an `h1` for example)

Reply:
The idea of using slots here is to leave open the possibility to add attributes (such as accessibility ones) and semantic tags (`<em>`, `<strong>`, `<sup>`, `<sub>`, ...). I think some product may need it